indico/web/flask/app.py

```diff
@@ -219,14 +219,15 @@ def setup_jinja(app):
     app.add_template_global(LocalProxy(lambda: current_plugin.manifest if current_plugin else None), 'plugin_webpack')
     # Useful constants
     app.add_template_global('^([0-9]|0[0-9]|1[0-9]|2[0-3]):[0-5][0-9]$', name='time_regex_hhmm')  # for input[type=time]
-    # Filters (indico functions returning UTF8)
+    # Filters
     app.add_template_filter(date_time_util.format_date)
     app.add_template_filter(date_time_util.format_time)
     app.add_template_filter(date_time_util.format_datetime)
     app.add_template_filter(date_time_util.format_human_date)
     app.add_template_filter(date_time_util.format_timedelta)
+    app.add_template_filter(date_time_util.format_interval)
```

Comment:
I just realized that this takes two values... so probably it shouldn't be a filter since it's really weird to pass the start time and end date in completely different ways...